.circleci/config.yml

```diff
@@ -86,6 +90,21 @@ jobs:
               --token "${GITHUB_RELEASES_TOKEN}" \
               --repo-url "${CIRCLE_REPOSITORY_URL}" \
 
+  update-lockfile:
+    description: Update lockfile in a release please PR
+    executor: node
+    steps:
+      - run: git clone -b "release-please--branches--main" "$CIRCLE_REPOSITORY_URL" --depth=2
```

Comment:
can you use the checkout action and instead just run it on the branch?

Reply:
This is the recommended way of making a custom checkout [docs](https://support.circleci.com/hc/en-us/articles/115015659247-How-do-I-modify-the-checkout-step-)

Reply:
But you don't need a custom checkout this should only run on this branch?